src/main/java/gregtech/loaders/recipe/MetaTileEntityMachineRecipeLoader.java

```diff
@@ -562,38 +563,56 @@ public static void init() {
                 .input(HULL[EV])
                 .input(frameGt, Titanium, 4)
                 .input(circuit, Tier.EV, 4)
-                .input(ELECTRIC_MOTOR_EV, 4)
+                .input(ELECTRIC_PISTON_EV, 4)
```

Comment:
Don't forget we have IV mining drills and it only takes minute to wipe a ore vein
If we nerf large miner, I guess nobody will use it...

Reply:
i do not think the power of the IV drill would detract people from using the nerfed large miners